src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -2092,10 +2030,7 @@ private void DoRemove(X509Certificate2 cert, bool fDeleteKey, bool fMachine, str
                     CommitUserDS(context.hCertStore);
                 }
 
-                if (DownLevelHelper.LogCertChangeSupported())
-                {
-                    Security.NativeMethods.LogCertDelete(fMachine, cert.Handle);
-                }
+                //TODO: Log Cert Delete
```

Comment:
I'm not sure we need these, but I'll open an issue or remove them based on my investigation.